components/local-app/cmd/root.go

```diff
@@ -92,13 +92,16 @@ var rootCmd = &cobra.Command{
 		}
 		cmd.SetContext(config.ToContext(context.Background(), cfg))
 
+		host := "https://gitpod.io"
 		telemetryEnabled := !telemetry.DoNotTrack()
 		telemetryEnabled = telemetryEnabled && cfg.Telemetry.Enabled
 		// For now we only enable telemetry on gitpod.io
-		if gpctx, err := cfg.GetActiveContext(); err == nil && gpctx != nil {
-			telemetryEnabled = telemetryEnabled && gpctx.Host.String() == "https://gitpod.io"
+		gpctx, err := cfg.GetActiveContext()
+		if err == nil && gpctx != nil {
+			telemetryEnabled = telemetryEnabled && gpctx.Host.String() == host
```

Comment:
> Make sure we can get some basic telemetry from Dedicated Instances as well.

This means we only get telemetry for Gitpod Cloud context

Reply:
You're absolutely correct 🙃

Reply:
We make all telemetry traffic to go through our proxy, then it's safe to only check the env, because Gitpod Instance will know if it needs to send telemetry.

But in Desktop Extension, we will pass every telemetry a `gitpodHost` property, to double check, maybe no need to do it the same in CLI